src/loaders.py

```diff
@@ -88,10 +89,12 @@ def __getitem__(self, index):
 
         if self.y is not None:
             Mi = self.y[0][index]
+            max_class = from_pil(Mi)[0].max()
 
             if self.train_mode and self.image_augment_with_target is not None:
                 Xi, Mi = from_pil(Xi, Mi)
                 Xi, Mi = self.image_augment_with_target(Xi, Mi)
+                Mi = np.where(Mi > max_class, max_class, Mi)
```

Comment:
Another solution would be to drop augmentations with target.

Reply:
@taraspiotr let's sleep on this one. There should be a nicer way to do it